ui/src/app/users/asideForm.tsx

```diff
@@ -0,0 +1,105 @@
+                            <label className="p-form__label">Password</label>
+                            <button className="p-button--base u-no-margin--bottom has-icon" style={{ float: "right" }} aria-live="polite" aria-controls="password" onClick={(e) => { e.preventDefault(); setShowPassword1(!showPassword1) }}>
+                                {showPassword1 ? (
+                                    <>
+                                        <span className="p-form-password-toggle__label">
+                                            Hide
+                                        </span>
+                                        <i className="p-icon--hide"></i>
+                                    </>
+                                ) : (
+                                    <>
+                                        <span className="p-form-password-toggle__label">
+                                            Show
+                                        </span>
+                                        <i className="p-icon--show"></i>
+                                    </>
+                                )}
+                            </button>
+                        </div>
+                        <input className="p-form-validation__input" type={showPassword1 ? "text" : "password"} id="password1" name="password" placeholder="******" autoComplete="current-password" required={true} onChange={handlePassword1Change} />
+                        <p className="p-form-help-text">
+                            Password must have 8 or more characters, must include at least one capital letter, one lowercase letter, and either a number or a symbol.
+                        </p>
+                        <label htmlFor="p-form__label">Confirm Password</label>
+                        <input className="p-form-validation__input" type="password" id="InputPassword" name="password2" placeholder="******" autoComplete="current-password" onChange={handlePassword2Change} />
+                        {!passwordIsValid(password1) && password1 != "" && <p className="p-form-validation__message">Password is not valid</p>}
+                        {passwordIsValid(password1) && !passwordsMatch && password2 != "" && <p className="p-form-validation__message">Passwords do not match</p>}
+                        {errorText &&
+                            <div className="p-notification--negative">
+                                <div className="p-notification__content">
+                                    <h5 className="p-notification__title">Error</h5>
+                                    <p className="p-notification__message">{errorText.split("error: ")}</p>
+                                </div>
+                            </div>
+                        }
+                        {!passwordsMatch || !passwordIsValid(password1) ? (
+                            <>
+                                <button type="submit" name="submit" disabled={true}>Submit</button>
```

Comment:
Can we make the submit button green (for consistency)?

Reply:
done